lib/from_erlang.ml

```diff
@@ -364,11 +364,15 @@ let code_to_module (F.AbstractCode form) =
 
 let module_to_expr m =
   let funs =
-    m.functions |> List.map ~f:(fun {specs; fun_name=name; args; body} ->
-                              (name, Abs (args, body)))
+    m.functions |> List.map ~f:(fun {specs; fun_name; fun_abst} -> (fun_name, fun_abst))
   in
-  let fun_names = funs |> List.map ~f:fst in
-  Letrec(funs, Tuple (List.map ~f:(fun name -> Var name) fun_names))
+  let body =
+    funs
+    |> List.map ~f:(fun (name, ({args; body}: fun_abst)) ->
+                  Tuple [Constant (Atom name); LocalFun {function_name=name; arity=List.length args}])
+    |> (fun es -> Tuple es)
```

Comment:
The reason for this is to make it easier to debug, so that you can see the correspondence between function names and types.

This change is less necessary.

Reply:
I got it. Thanks!